coq_tools/find_bug.py

```diff
@@ -2247,7 +2247,10 @@ def make_make_coqc(coqc_prog, **kwargs):
                         list(env[passing_prefix + "libnames"]) + [(coq_theories_path, "Coq")]
                     )
                     env[passing_prefix + "libnames"] = tuple(
-                        list(env[passing_prefix + "libnames"]) + [(coq_user_contrib_path, "Coq")]
```

Comment:
Is this change actually necessary at this point?  It breaks minimization on old versions of Coq.  Maybe you meant to replace Stdlib with Coq below?